R/kernel.r

```diff
@@ -312,6 +316,8 @@ initialize = function(connection_file) {
 
     executor <<- Executor$new(send_response = .self$send_response,
         abort_queued_messages = .self$abort_queued_messages)
+    comm_manager <<- Comm_Manager$new(send_response = .self$send_response)
+    assign('kernel', .self, envir = .GlobalEnv)
```

Comment:
this is related to #24.

if we do it this way, a valid check would be `inherits(.GlobalEnv$kernel, 'kernel')`.

another possibility would be to have a global kernel instance in the `IRkernel` namespace, which is usually `NULL` unless we run in a kernel.

maybe better than modifying the global env, what do y’all think?